upup/models/cloudup/resources/addons/storage-gce.addons.k8s.io/v1.6.0.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: storage.k8s.io/v1beta1
+kind: StorageClass
+metadata:
+  name: standard
+  annotations:
+    storageclass.beta.kubernetes.io/is-default-class: "true"
```

Comment:
Storage class changed API in 1.7. We need on in for both aws and GCE. We can do another pr